app/tests/spec/models/password_strength/password_strength_balloon.js

```diff
@@ -90,8 +91,11 @@ describe('models/password_strength/password_strength_balloon', () => {
       'password',
       'firefox account',
       'firefox accounts',
-      'FirefoxAccounts', 'firefox sync',
+      'FirefoxAccounts',
+      'firefox sync',
       'Firefox   SYNC',
+      'mozilla123',
+      '123mozilla', // mozilla doesn't have to be at the start, but comprises over half the password
```

Comment:
Does anything in this test assert that the requirement for service names to be sorted in descending order of length is correctly in place? It doesn't look like it from what I can make out. If it's important, should we add something?

Reply:
> Does anything in this test assert that the requirement for service names to be sorted in descending order of length is correctly in place?

Hmm... I can put the ["firefox" before "firefox account"](https://github.com/mozilla/fxa-content-server/pull/6341/files#diff-b0058b670fab14faa421cedbb199595aR19) and the password "firefox account" would be allowed.